policy/modules/services/dovecot.if

```diff
@@ -61,6 +61,85 @@ interface(`dovecot_domtrans_deliver',`
 	domtrans_pattern($1, dovecot_deliver_exec_t, dovecot_deliver_t)
 ')
 
+########################################
+## <summary>
+##	Read and write dovecot unix
+##	stream sockets.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`dovecot_rw_stream_sockets',`
+	gen_require(`
+		type dovecot_t;
+	')
+
+	allow $1 dovecot_t:unix_stream_socket { connectto rw_stream_socket_perms };
+')
+
+########################################
+## <summary>
+##	Read and write dovecot runtime
+##	sock files.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`dovecot_rw_runtime_sock_files',`
+	gen_require(`
+		type dovecot_runtime_t;
+	')
+
+    rw_sock_files_pattern($1, dovecot_runtime_t, dovecot_runtime_t)
+')
+
+########################################
+## <summary>
+##	Read dovecot configuration content.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+## <rolecap/>
+#
+interface(`dovecot_read_config',`
+	gen_require(`
+		type dovecot_etc_t;
+	')
+
+	files_search_etc($1)
+	allow $1 dovecot_etc_t:dir list_dir_perms;
+	allow $1 dovecot_etc_t:file read_file_perms;
+	allow $1 dovecot_etc_t:lnk_file read_lnk_file_perms;
+')
+
+########################################
+## <summary>
+##	Search dovecot runtime content.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+## <rolecap/>
+#
+interface(`dovecot_search_runtime',`
+	gen_require(`
+		type dovecot_runtime_t;
+	')
+
+	search_dirs_pattern($1, dovecot_runtime_t, dovecot_runtime_t)
+')
```

Comment:
this one is redundant and can be removed as `dovecot_rw_runtime_sock_files` implies this access